Gemfile.lock

```diff
@@ -111,6 +111,7 @@ GEM
       rb_sys (~> 0.9)
     selma (0.4.11-aarch64-linux)
     selma (0.4.11-arm64-darwin)
+    selma (0.4.11-x86_64-darwin)
```

Comment:
TODO: run `bundle cache` so the cache is updated with this `.gem` https://github.com/rubygems/rubygems/pull/7896#issuecomment-2582506686